geometry_msgs/Point robot_pos

---

float64 probability
